### ISSUE TITLE
Feature Request: Highlight one tag

### ISSUE THREAD

**Issue by homersimpsons** [openstreetmap/iD#3137]
Hello,

I think this could be usefull to highlight one specific tag, especially by displaying it where it is. Why?
Because, I started to map "lit=yes" in some place, but now it's harder to find which road or road segment that haven't got that tag. I started also to map "lanes" and "turn:lanes" and, same as before, I can't easily view which ones are not tagged.

**Comment by bhousel** [comment]
Thanks for the request.. I agree this would be really useful if we can find a place for it.  This is one of those things that might not belong in iD, but we can leave the issue open for discussion.

Not sure if you've seen it but, @tyrasd's [Overpass Turbo](http://overpass-turbo.eu/) is really great at this.  You can query for tags like that and send the results to JOSM via remote control.

**Comment by homersimpsons** [comment]
Yes I know about Overpass but it is really much longer than getting it directly in iD
